src/api/java/mekanism/api/MekanismIMC.java

```diff
@@ -85,8 +87,25 @@ public static void addModuleContainer(ModuleContainerTarget moduleContainer) {
      *
      * @apiNote Call this method during the {@link InterModEnqueueEvent}.
      * @since 10.5.0
+     * @deprecated Call via {@link #addModulesToAll(Holder[])}
```

Comment:
this should be `Use {@link #X} instead` so allow IDEA's heuristic intention to work best